src/basics.md

```diff
@@ -142,7 +146,7 @@ In the meantime we recommend to do as they did in the olden days and return an e
 
 ### Closures
 
-Closures are not yet supported and we are waiting for the [Function References](https://github.com/WebAssembly/function-references) 🦄 proposal (`func.bind`?) to land. However, since this is a crucial language feature, work on a preliminary implementation not depending on future WebAssembly features [has started](https://github.com/AssemblyScript/assemblyscript/pull/1308), but the prototype is still in flux and currently limited to read only captures. You can try it out with `npm install assemblyscript-closures-beta`.
+Closures are not yet supported and we are waiting for the [Function References](https://github.com/WebAssembly/function-references) 🦄 proposal (`func.bind`?) to land. 
```

Comment:
Are we still going to wait for Function References before starting closures? :smile:

Reply:
Removed the paragraph because the `assemblyscript-closures-beta` branch is kinda outdated with the runtime rework in place. Otherwise doesn't imply anything new :)

Reply:
Re-added a reference that we may up with something custom earlier.